build.gradle

```diff
@@ -4,10 +4,10 @@ buildscript {
   ext.versions = [
     minSdk: 23,
     compileSdk: 33,
-    kotlin: '1.7.20',
-    composeUi: '1.3.1',
-    composeCompiler: '1.4.0-alpha01',
-    composeUiMaterial3: '1.0.1',
+    kotlin: '1.8.21',
+    composeUi: '1.5.0-alpha04',
```

Comment:
Let's depend on `1.5.0-alpha01`. Not all projects may be ready to consume newer alpha versions yet.